clio-client/src/test/scala/org/broadinstitute/clio/client/dispatch/MoveWgsUbamSpec.scala

```diff
@@ -4,11 +4,10 @@ import org.broadinstitute.clio.client.BaseClientSpec
 import org.broadinstitute.clio.client.commands.MoveWgsUbam
 import org.broadinstitute.clio.client.util.MockIoUtil
 import org.broadinstitute.clio.client.webclient.MockClioWebClient
-import org.broadinstitute.clio.transfer.model.TransferWgsUbamV1Key
 import org.broadinstitute.clio.util.model.Location
-
 import akka.http.scaladsl.model.StatusCodes
 import akka.http.scaladsl.model.headers.OAuth2BearerToken
+import org.broadinstitute.clio.transfer.model.wgsubam.TransferWgsUbamV1Key
 
 class MoveWgsUbamSpec extends BaseClientSpec {
```

Comment:
Once we have all operations generalized we no longer need to provide a test spec for each different document type, as the code is the same.

Instead we'd want a test for `AddExecutor`, `MoveExecutor`, etc. It would be nice if it could be tested independently of `ClioWebClient` too.